nautobot_golden_config/models.py

```diff
@@ -492,29 +489,23 @@ class GoldenConfigSetting(PrimaryModel):
 
     def get_absolute_url(self):  # pylint: disable=no-self-use
         """Return absolute URL for instance."""
-        return reverse("plugins:nautobot_golden_config:goldenconfigsetting")
+        return reverse("plugins:nautobot_golden_config:goldenconfigsetting", args=[self.slug])
 
     def __str__(self):
         """Return a simple string if model is called."""
-        return "Configuration Object"
-
-    def delete(self, *args, **kwargs):
-        """Enforce the singleton pattern, there is no way to delete the configurations."""
+        return f"Golden Config Setting - {self.name}"
 
     class Meta:
-        """Set unique fields for model."""
+        """Set unique fields for model.
 
-        verbose_name = "Golden Config Setting"
+        Provide ordering used in tables and get_device_to_settings_map.
+        """
 
-    @classmethod
-    def load(cls):
-        """Enforce the singleton pattern, fail it somehow more than one instance."""
-        if len(cls.objects.all()) != 1:
-            raise ValidationError("There was an error where more than one instance existed for a setting.")
-        return cls.objects.first()
+        verbose_name = "Golden Config Setting"
+        ordering = ["-weight", "name"]
```

Comment:
I would add a comment here on why `-weight`, to those in the future, it may seem odd why not order by name/slug.